### ISSUE TITLE
Link up failed Debian 10 installation

### ISSUE THREAD

**Issue by Erriez** [eaksel/packer-Debian10#7]
Ethernet link does not come up after first boot imported OVF in VirtualBox.

@eaksel In the mean time I found the root cause and can provide a MR, but Issues #1 and #2 needs to be merged first, otherwise I'll go ahead with my own project. Thanks.

**Comment by eaksel** [comment]
Hi @Erriez 
You can go ahead and send a PR, I've merged the two PR you've sent.

If you'd like to create your own repo please feel free to do so and don't hesitate to use the files that are on this repo, there's no need to mention me.
I can also add you as a collaborator if you'd like to.

**Comment by Erriez** [comment]
@eaksel Thanks for merging the other PR. Good to see you're maintaining this project. I'll create a new PR for you with fixes in the same files. Enabling ACPI/IOAPIC VirtualBox configuration fixes the problem.
